scripts/Tools/xmlchange

```diff
@@ -1,13 +1,49 @@
 #!/usr/bin/env python
 
 """
- This utility allows the user to change a env_*xml file via a commandline interface.
-
- The command is echoed to the CaseStatus file, unless -noecho is given. The
- purpose of this echoing is to provide a "paper trail" of changes made by the
- user, so calls to xmlchange by the cime scripts that are part of the normal case
- setup/build process should generally use -noecho.
-
+Allows changing variables in env_*xml files via a command-line interface.
+
+This provides two main benefits over editing the xml files by hand:
+  - Settings are checked immediately for validity
+  - Settings are echoed to the CaseStatus file, providing a "paper trail" of
+    changes made by the user.
+
+Examples:
```

Comment:
I tried to make the xmlchange help roughly parallel the help for xmlquery, though I didn't go so far as to make the help text for these two tools completely consistent (mainly because I ran out of time and energy).